policies/stable-release-updates.md

```diff
@@ -6,8 +6,9 @@ This policy covers allowed changes on stable release branches.
 Definitions
 -----------
 
-A **stable release** is a patch release from an existing supported minor
-release branch.
+A **stable release** is a major or minor release that is not a pre-release.
+
+A **patch release** is an update of a stable release.
```

Comment:
The wording implies that a patch release is not a stable release itself - just an update of a stable release. Could that be made clearer somehow?

It would also be useful to have standard terminology to refer to the whole series of releases rather than an individual patch level. I often find myself talking about the 3.0 release (where I mean any of 3.0.0, 3.0.1, 3.0.2 etc), and also the 3.0.1 release where I'm talking about a specific patch level.

Reply:
Maybe call that a **stable release series**?

Reply:
Possibly. Although that's a bit of a mouthful. Maybe just "release series". Or even just "series".

Reply:
I've used "release update" for things like 1.1.1m or 3.0.1.  We have used "stable release" a bit vaguely, I'm never sure if we mean 3.0.0 or the whole series of 3.0.x...  I guess that's basically what @mattcaswell describes.  I frankly do not care what we end up calling it, but I would say that taking away "stable release" wouldn't be very good, since this whole document is a **stable release *policy***

Reply:
I could go with just defining "stable release" as being shorthand for "stable release series" which means the whole set 3.0.x. So you could talk about the 3.0 stable release or the 1.1.1 stable release.